src/main/cli/settings.c

```diff
@@ -1597,7 +1597,8 @@ const clivalue_t valueTable[] = {
 #endif
 
 // PG_POSITION
-    { "position_alt_source",           VAR_INT8   | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_POSITION_ALT_SOURCE }, PG_POSITION, offsetof(positionConfig_t, altSource) },
+    { "position_alt_source",           VAR_INT8   | MASTER_VALUE  | MODE_LOOKUP, .config.lookup = { TABLE_POSITION_ALT_SOURCE }, PG_POSITION, offsetof(positionConfig_t, altSource) },
+    { "level_race_mode",               VAR_UINT8  | PROFILE_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_OFF_ON }, PG_PID_PROFILE, offsetof(pidProfile_t, level_race_mode) },
```

Comment:
These settings are ordered by parameter group - a new setting for `PG_PID_PROFILE` should go to the bottom of the existing `PG_PID_PROFILE` block to make future maintenance easier.

Reply:
Thanks, moved!